app/models/user.rb

```diff
@@ -14,6 +14,9 @@ class User < ApplicationRecord
   before_save :downcase_email
   before_create :create_activation_digest
 
+  scope :activated, ->{where activated: true}
+  scope :search, ->(term){where "name LIKE ?", "%#{term}%"}
```

Comment:
```suggestion
  scope :search_by_name, ->(term){where "name LIKE ?", "%#{term}%" if term.present? }
```
để thế này nếu term trống sẽ trả về all bản ghi